pkg/kubelet/cm/container_manager_windows.go

```diff
@@ -369,3 +369,16 @@ func (cm *containerManagerImpl) UnprepareDynamicResources(ctx context.Context, p
 func (cm *containerManagerImpl) PodMightNeedToUnprepareResources(UID types.UID) bool {
 	return false
 }
+
+func (cm *containerManagerImpl) HasExclusiveCPUs(pod *v1.Pod) bool {
+	for _, container := range pod.Spec.Containers {
```

Comment:
Should this also be hidden behind the `WindowsCPUAndMemoryAffinity` feature flag as some of the other code in this file is?

Reply:
I think this is unnecessary because if the  `WindowsCPUAndMemoryAffinity` is disabled, then we transparently use the FakeCPUManager which will always return empty for exclusive assignment. We waste some cpu cycles running the loop anyway, but this should not be a big deal